testnet/stacks-node/src/syncctl.rs

```diff
@@ -416,7 +426,8 @@ impl PoxSyncWatchdog {
         burnchain_tip: &BurnchainTip, // this is the highest burnchain snapshot we've sync'ed to
         burnchain_height_opt: Option<u64>, // this is the absolute burnchain block height, if known
         num_sortitions_in_last_cycle: u64,
-    ) -> bool {
+        should_keep_running: Arc<AtomicBool>,
```

Comment:
This argument should not be required anymore, because it's now a member of this struct.

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/2547/commits/7354a9ca4677aea340d4e538eee5aa38f645a66f